nautobot/dcim/models/devices.py

```diff
@@ -1456,6 +1467,11 @@ class ControllerManagedDeviceGroup(TreeModel, PrimaryModel):
         null=False,
         help_text="Controller that manages the devices in this group",
     )
+    capabilities = JSONArrayField(
+        base_field=models.CharField(choices=ControllerCapabilitiesChoices),
+        blank=True,
+        null=True,
+    )
```

Comment:
Was there a reason this wasn't being inherited from the controller? Was it because we wanted to support a use case where a controller supports "X and Y" but the managed group only supports "X"? Have we identified any controllers that would have multiple capabilities?

Reply:
Right now we only have 1 targeted capability. @lampwins Would you like to weigh in?